src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,13 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+
+    private static Random random = new Random();
```

Comment:
Avoid using static variables for mutable objects. Instead, you can instantiate a new Random object within the method where it's needed or as an instance variable if it's shared within the class.